java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileInfoStore.java

```diff
@@ -118,7 +111,7 @@ public void addFile(FileInfo fileInfo) throws StateStoreException {
 
     public void addFile(FileInfo fileInfo, long updateTime) throws StateStoreException {
         try {
-            String tableName = tableName(fileInfo);
+            String tableName = activeTableName;
```

Comment:
It might be a bit clearer if we remove this local variable. It's easy to think this might be the Sleeper table name.